source/libsys/source/Guard.cc

```diff
@@ -87,4 +93,26 @@ namespace sys::env {
         }
         return result;
     }
+
+    const char** get_envp()
+    {
+#ifdef HAVE_NSGETENVIRON
+        return const_cast<const char**>(*_NSGetEnviron());
+#else
+        return const_cast<const char**>(environ);
+#endif
+    }
+
+    const Vars& get()
+    {
```

Comment:
This method does not look good to me! :smile: Static variables are source of bugs in my books.

The static variables can be avoided by change the return type by value (instead of by reference). You can still can capture the return value as `const auto &` to avoid copying. We shall just rely on compiler return value optimization then declaring static variables.